docs/sources/tutorials/alerting-get-started-pt3/index.md

```diff
@@ -0,0 +1,446 @@
+
+   <!-- INTERACTIVE exec START -->
+
+   ```
+   cd tutorial-environment
+   ```
+
+   <!-- INTERACTIVE exec END -->
+
+1. Run the Grafana stack:
+
+   <!-- INTERACTIVE ignore START -->
+
+   ```
+   docker compose up -d
+   ```
+
+   <!-- INTERACTIVE ignore END -->
+
+   {{< docs/ignore >}}
+
+   <!-- INTERACTIVE exec START -->
+
+   ```bash
+   docker-compose up -d
+   ```
+
+   <!-- INTERACTIVE exec END -->
+
+   {{< /docs/ignore >}}
+
+   The first time you run `docker compose up -d`, Docker downloads all the necessary resources for the tutorial. This might take a few minutes, depending on your internet connection.
+
+   <!-- INTERACTIVE ignore START -->
+
+   {{< admonition type="note" >}}
+   If you already have Grafana, Loki, or Prometheus running on your system, you might see errors, because the Docker image is trying to use ports that your local installations are already using. If this is the case, stop the services, then run the command again.
+   {{< /admonition >}}
+
+   <!-- INTERACTIVE ignore END -->
+
+   {{< docs/ignore >}}
+
+   NOTE:
+
+   If you already have Grafana, Loki, or Prometheus running on your system, you might see errors, because the Docker image is trying to use ports that your local installations are already using. If this is the case, stop the services, then run the command again.
+
+   {{< /docs/ignore >}}
+
+<!-- INTERACTIVE page step1.md END -->
+<!-- INTERACTIVE page step2.md START -->
+
+## How alert rule grouping works
+
+Alert notification grouping is configured with **labels** and **timing options**:
+
+- **Labels** map the alert rule with the notification policy and define the grouping.
+- **Timing options** control when and how often notifications are sent.
+
+{{< figure src="/media/docs/alerting/alerting-notification-policy-diagram-with-labels-v3.png" max-width="750px" alt="A diagram about the components of a notification policy, including labels and groups" >}}
+
+### Types of Labels
+
+1. **Reserved labels** (default):
+
+   - Automatically generated by Grafana, e.g., `alertname`, `grafana_folder`.
+   - Example: `alertname="High CPU usage"`.
+
+1. **User-configured labels**:
+
+   - Added manually to the alert rule.
+   - Example: `severity`, `priority`.
+
+1. **Query labels**:
+   - Returned by the data source query.
+   - Example: `region`, `service`, `environment`.
+
+### Timing Options
+
+1. **Group wait**: Time before sending the first notification.
+1. **Group interval**: Time between notifications for a group.
+1. **Repeat interval**: Time before resending notifications for an unchanged group.
+
+Alerts sharing the **same label values** are grouped together, and timing options determine notification frequency.
+
+For more details, see:
+
+- [Grouping Alerts](ref:alert-grouping)
+- [Alert Labels](ref:alert-labels)
+
+<!-- INTERACTIVE page step2.md END -->
+<!-- INTERACTIVE page step3.md START -->
+
+## A real-world example of alert grouping in action
```

Comment:
you need to add some text under this header - like an intro line. we don't have sequential headers